docs/src/jekyll/_plugins/include_remote.rb

```diff
@@ -0,0 +1,48 @@
+require 'nokogiri'
+require 'open-uri'
+require 'uri'
+
+class Jekyll::IncludeRemoteTag < Jekyll::Tags::IncludeTag
```

Comment:
I wonder if there is a way to provide all of of these includes also so that projects don't have to include jekyll sources themselves. Not related to this PR though, I know all projects have it.